fvm/evm/handler/handler.go

```diff
@@ -227,6 +347,22 @@ func (h *ContractHandler) checkGasLimit(limit types.GasLimit) error {
 	return nil
 }
 
+// decodeTransaction decodes RLP encoded transaction payload and meters the resources used.
+func (h *ContractHandler) decodeTransaction(encodedTx []byte) (*gethTypes.Transaction, error) {
```

Comment:
Can we use this method for the single run as well ?

Reply:
Good idea